pom.xml

```diff
@@ -139,6 +140,28 @@
     </pluginManagement>
 
     <plugins>
+      <!-- Strip any qualifiers from version and expose the result as release.version property -->
+      <plugin>
+        <groupId>org.codehaus.mojo</groupId>
+        <artifactId>build-helper-maven-plugin</artifactId>
+        <version>${plugin.build-helper-maven-version}</version>
+        <executions>
+          <execution>
+            <id>set-release-version</id>
+            <goals>
+              <goal>parse-version</goal>
+              <goal>regex-property</goal>
+            </goals>
+            <configuration>
+              <name>release.version</name>
+              <regex>.</regex>
+              <value>-</value>
+              <!--suppress UnresolvedMavenProperty -->
+              <replacement>${parsedVersion.majorVersion}.${parsedVersion.minorVersion}.${parsedVersion.incrementalVersion}</replacement>
+            </configuration>
+          </execution>
```

Comment:
This plugin gives us the ability to get the `release.version` as a maven / ant property and also to get it from commandline with:
`mvn validate help:evaluate -Dexpression=release.version -q -DforceStdout`